sources.list

```diff
@@ -1,27 +1,27 @@
 # See http://help.ubuntu.com/community/UpgradeNotes for how to upgrade to
 # newer versions of the distribution.
 
-deb http://archive.ubuntu.com/ubuntu/ trusty main restricted
-# deb-src http://archive.ubuntu.com/ubuntu/ trusty restricted main multiverse universe
+deb http://archive.ubuntu.com/ubuntu/ xenial main restricted
```

Comment:
We will fix this is in a separate issue then! [There's already one ;)] @yukiisbored